testdata/project-v2-addon/go.mod

```diff
@@ -9,5 +9,5 @@ require (
 	k8s.io/apimachinery v0.18.4
 	k8s.io/client-go v0.18.4
 	sigs.k8s.io/controller-runtime v0.6.0
-	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20200816135617-dbfe418e405f
+	sigs.k8s.io/kubebuilder-declarative-pattern v0.0.0-20201008142216-045672be83e7
```

Comment:
For master, we pinned the version now. However, for the book-v2 it still getting the latest. In this way, for we be able to update the docs until the next release we need to fix it.